resources/assets/js/components/accepted-topics.vue

```diff
@@ -17,6 +17,8 @@
                             Covered/will cover in episode:
                             <topic-episode-scheduler :topic="topic" :episodes="episodes"></topic-episode-scheduler>
                         </div>
+                        Link:
+                        <topic-link :topic="topic"></topic-link>
```

Comment:
If we're going to make anything editable on the topic, I'd prefer we make the entire thing editable--and then this URL would be one of the properties that you can set both when you create and edit a topic.